clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -1967,9 +1942,13 @@ private void validateValueSchema(
     }
     if (getValueSchemaIdResponse.isError() && !schemaAutoRegisterFromPushJobEnabled) {
       MultiSchemaResponse response = controllerClient.getAllValueSchema(setting.storeName);
-      LOGGER.info("All currently registered value schemas:");
-      for (MultiSchemaResponse.Schema schema: response.getSchemas()) {
-        LOGGER.info("Schema " + schema.getId() + ": " + schema.getSchemaStr());
+      if (response.isError()) {
+        LOGGER.error("Failed to fetch all value schemas, so they will not be printed.");
+      } else {
+        LOGGER.info("All currently registered value schemas:");
+        for (MultiSchemaResponse.Schema schema: response.getSchemas()) {
+          LOGGER.info("Schema " + schema.getId() + ": " + schema.getSchemaStr());
```

Comment:
Let's use interpolation format for logging here

Reply:
Done, thanks!